README.md

```diff
@@ -72,10 +72,16 @@ A `json` configuration file of following format is used to configure Grabbit.
     "serverPort" : "4502",
     "pathConfigurations" :  [
         {
-            "path" : "/content/someContent"
+            "path" : "/content/someContent",
+            "excludePaths" : "someContent/someExcludeContent"
         },
         {
             "path" : "/content/dam/someDamContent",
+            "excludePaths":
+                [
+                    "someContent/someExcludeContent",
+                    "someContent/someOtherExcludeContent"
+                ],
```

Comment:
Where is the description of what "excludePaths" means and what rules there are around it?